### PR TITLE
Added functionality for bot to search for recipies

### LINKED ISSUE #13: add weekly gluten picture/recipe, or !recipe command

### PR DESCRIPTION
**Pull request by Chippers255**
This satisfies a part of issue #13. The !recipe command will now exist on all channels allowing people to search allrecipes.

### CODE REVIEW

**Comment by twentylemon** [comment]
Please fix lint errors. See [checks](https://github.com/Chippers255/duckbot/runs/1990475953?check_suite_focus=true). Also fix the code smells sonarcloud is complaining about.

File: duckbot/cogs/recipe.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+from discord.ext import commands
+
+
+class Recipe(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @staticmethod
+    def search(search_term):
```
Comment (twentylemon) [inline duckbot/cogs/recipe.py]: This method should be broken down, it's doing a lot of work. I'd probably do three methods to make this easier to understand:
* fetch recipes for a search term
* select a single recipe given a collection
* send a recipe card to the channel

**Review by twentylemon — CHANGES_REQUESTED** [review]
Good stuff overall.

**Comment by Chippers255** [comment]
Marked as draft, I need to fix the linting errors and write a couple more tests on the command method.

**Review by twentylemon — APPROVED** [review]
Good stuff, brother.

**Pull request merged**